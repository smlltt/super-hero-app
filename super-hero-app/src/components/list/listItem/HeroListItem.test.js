import React from "react";
import renderer from "react-test-renderer";
import HeroListItem from "./index";

it("changes arrow to cursor on hover", () => {
  const tree = renderer
    .create(
      <HeroListItem hero={{ name: "Batman", image: { url: "example.jpg" } }} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
