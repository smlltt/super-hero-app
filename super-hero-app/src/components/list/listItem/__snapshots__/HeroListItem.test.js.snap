// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes arrow to cursor on hover 1`] = `
<li
  className="MuiListItem-root sc-gtsrHT bpCfxK MuiListItem-gutters"
  disabled={false}
  onClick={[Function]}
>
  <div
    className="MuiListItemText-root"
  >
    <span
      className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
    >
      Batman
    </span>
  </div>
  <div
    className="MuiListItemAvatar-root"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circle"
    >
      <img
        alt="Batman"
        className="MuiAvatar-img"
        src="example.jpg"
      />
    </div>
  </div>
</li>
`;
